### PR TITLE
Lesson20-Kadir

### PR DESCRIPTION
**Pull request by Kadir-CS**
1st attempt

### CODE REVIEW

File: Lesson20/Kadir/app.js [hunk 1]
```diff
@@ -9,25 +9,51 @@ form.addEventListener('submit', (event) => {
...
+  spanElement.innerText = input.value;
+  listItemElement.appendChild(spanElement);
   
   // Remove button
   const removeButton = document.createElement("button");
   removeButton.innerText = 'X';
   // TODO: Update styles of removeButton! (use button--light class)
-  listItemElement.appendChild(removeButton);
-
+
```
Comment (annasvst) [inline Lesson20/Kadir/app.js:20]: TODO: Update styles of removeButton! (use button--light class)TODO: Update styles of removeButton! (use button--light class)

This requirement is still missing, please add styles to the delete button

File: Lesson20/Kadir/app.js [hunk 2]
```diff
@@ -9,25 +9,51 @@ form.addEventListener('submit', (event) => {
...
   list.appendChild(listItemElement);
 
   // Add a listener to the task item to toggle its completed status
-  listItemElement.addEventListener('click',() => {
+  spanElement.addEventListener('click',() => {
     // TODO: "completed" class is applied both to the text of the item and the button. Refactor the content of li element so that the class is only applied to the text but not to the button!
-    listItemElement.classList.toggle("completed");
+
+   
+    // listItemElement.classList.toggle("completed");
```
Comment (annasvst) [inline Lesson20/Kadir/app.js:28]: In real life, you should not be allowed to leave comments like this in the code, please remove all the code that is not used. You should always be able to find valuable things in the Git history when looking for the commits made in this PR, for example

File: Lesson20/Kadir/app.js (hunk 1 above)
Comment (Kadir-CS) [reply Lesson20/Kadir/app.js:20]: It  is in row 29 already.

**Review by annasvst — APPROVED** [review]
Approved!

**Pull request merged**